### DISCUSSION TITLE
How Do I Export My Toolbar Configuration?

### DISCUSSION THREAD

**Discussion by vladQll** [xournalpp/xournalpp#5044] [Q&A]
I was reviewing about the proposal of the redesign of xournal , then I liked that it looked minimalist , and compact , so I tried to imitate it , I spent a couple of hours , between configuring the bar , try to make the graphical tablet buttons work properly , configure the pen pressure among other things , at least I saved the plugin file in case I have to use it on a new computer , my question is how do I export all my toolbar configuration including the template background although this last one could be done in an instant, but what interests me the most is to know how do I save all that configuration or what file where all those configurations are applied I should copy to my backup folder.

![image](https://github.com/xournalpp/xournalpp/assets/28522011/a4403f3c-6e61-4fa1-9242-e896d36961dc)

**Comment by rolandlo** [comment]
Just backup your `config`-folder. See [file locations](https://xournalpp.github.io/guide/file-locations/)
